pages/windows/set-service.md

```diff
@@ -0,0 +1,17 @@
+# Set-Service
+
+> Starts, stops, and suspends a service, and changes its properties.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.management/set-service>.
+
+- Change a display name:
+
+`Set-Service -Name {{LanmanWorkstation}} -DisplayName {{"LanMan Workstation"}}`
```

Comment:
Is this a hostname?
```suggestion
`Set-Service -Name {{hostname}} -DisplayName "{{name}}"`
```

Reply:
To be consistent with other commands I suggest

```suggestion
`Set-Service -Name "{{hostname}}" -DisplayName "{{name}}"`
```

Reply:
who use spaces in host names?